cmd/fyne_demo/screens/window.go

```diff
@@ -172,6 +172,18 @@ func loadDialogGroup(win fyne.Window) *widget.Card {
 				fileSaved(writer)
 			}, win)
 		}),
+		widget.NewButton("Folder Open", func() {
+			dialog.ShowFolderOpen(func(list fyne.ListableURI, err error) {
+				if list == nil {
+					return
+				}
+				if err != nil {
+					dialog.ShowError(err, win)
+					return
+				}
```

Comment:
This ordering means the error message will not get shown

Reply:
Thanks, fixed